### DISCUSSION TITLE
Error when using a new CRS : Problem with the synchronization with the EPSG database

### DISCUSSION THREAD

**Discussion by mabkoz** [pyproj4/pyproj#1040] [General]
Hello community! 

I am writing a python script to convert coordinates from EPSG:4937 to EPSG:9895. I am facing an error which is that the EPSG:9895 is not a valid CRS " _Invalid projection: EPSG:9895 (Internal Proj Error: proj_create: crs not found)._ " . When I go to https://epsg.org/ , I can find the CRS in the EPSG Dataset : v10.058. Correct if I am wrong but I think that the pyproj does not take into consideration the latest version of the Dataset which is v10.058. 

**Could you please tell me how to force pyProj to synchronize with the latest version of EPSG Dataset in order to recognize my CRS ? Thank you in advance!** 

------------------------------------------
 
**Here is the code where I encounter the problem  :**

trans_group = TransformerGroup(CRS("EPSG:4937").to_3d(),CRS("EPSG:9895").to_3d())
(The first one is OK, the second one is not recognizable)

**Here is my installations :** 

proj-data-1.8 installed with Anaconda

**Comment by snowman2** [comment]
https://github.com/OSGeo/PROJ/issues/3138
